src/ctia/properties.clj

```diff
@@ -113,3 +113,6 @@
 (def init! (mp/build-init-fn files
                              PropertiesSchema
                              properties))
+
+(defn get-http-show []
+  (get-in @properties [:ctia :http :show]))
```

Comment:
It might be overkill to do it right now. But `get-http-show` will be called a lot and the properties won't change much often. Wouldn't it be a good idea to memoize it for at least few seconds?